### PR TITLE
feat(web-cloud.exchange): add DKIM configurator for no ovh cloud 

### PR DESCRIPTION
**Pull request by antonymarion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/dkim-b2`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat #MANAGER-12208
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [ ] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Enable to configure DKIM with domain outside ovh

### CODE REVIEW

File: packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.controller.js [hunk 1]
```diff
@@ -68,6 +97,115 @@ export default class ExchangeDomainDkimAutoconfigCtrl {
...
+          this.services.$translate.instant(
+            'exchange_tab_domain_diagnostic_dkim_configurate_no_ovhcloud_failed',
+          ),
+        );
+      });
+  }
+
+  initSpfContext() {
+    this.showSpfDiagnosticTitle = true;
+    this.getSelectorNameForNoOvhCloud();
```
Comment (ghyenne) [inline packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.controller.js:174]:
```suggestion
    return this.getSelectorNameForNoOvhCloud();
```

**Review by ghyenne — CHANGES_REQUESTED** [review]

File: packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.html [hunk 2]
```diff
@@ -34,4 +28,75 @@
             </div>
         </div>
     </div>
+
+    <!-- if unconfigured without OVHcloud domain -->
+    <div ng-if="ctrl.dkimForNoOvhCloud">
+        <div
+            data-wizard
+            data-wizard-on-finish="leaveDkimConfigurator"
+            data-wizard-on-cancel="leaveDkimConfigurator"
```
[the remark this answers is not in the document]
Comment (antonymarion) [reply packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.html:37]: Yes, basically, leaveDkimConfigurator reset all vars for modals stepper; and once configuration is done then chip will be red until user updates with given values from last modal (step 3 of configurator)

File: packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.controller.js [hunk 3]
```diff
@@ -68,6 +97,115 @@ export default class ExchangeDomainDkimAutoconfigCtrl {
...
+          selectorName: dkimSelector,
+          autoEnableDKIM: index === 0,
+          configureDkim: true,
+        },
+      );
+    });
+
+    return this.services.$q
+      .all(promises)
+      .then((res) => res)
```
Comment (darsene) [inline packages/manager/modules/exchange/src/domain/dkim-autoconfig/domain-dkim-autoconfig.controller.js:161]:
```suggestion
      
```
Seems useless

**Review by darsene — CHANGES_REQUESTED** [review]

**Review by antonymarion — COMMENTED** [review]
done

**Pull request closed without merging**